libnavui-voice/src/main/res/layout/mapbox_audio_guidance_button_layout.xml

```diff
@@ -11,9 +11,11 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/container"
         android:layout_width="wrap_content"
-        android:layout_height="52dp"
-        android:minWidth="72dp"
-        android:paddingStart="@dimen/mapbox_audioGuidanceButton_paddingStart">
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:paddingStart="10dp"
+        android:paddingTop="10dp"
+        android:paddingBottom="10dp">
```

Comment:
```suggestion
        android:paddingStart="@dimen/mapbox_extendableButton_paddingStart"
        android:paddingTop="@dimen/mapbox_extendableButton_paddingStart"
        android:paddingBottom="@dimen/mapbox_extendableButton_paddingStart"
```

Reply:
The `MapboxExtendableButton` class and all its dimen (`mapbox_extendableButton_*`) and style values are in the `libnavui-dropin` module are not accessible from the `libnavui-voice`.

Should I move `MapboxExtendableButton` and `MapboxCameraModeButton` to the `libnavui-base`?